### ISSUE TITLE
AdminShell updateObjectTemplates reports error when it finds no updates to perform

### ISSUE THREAD

**Issue by chrisinmtown** [MISP/MISP#6715]
Please repair the AdminShell's updateObjectTemplates function so it accurately says "nothing done" instead of  complaining that it could not update templates.

https://github.com/MISP/MISP/blob/ce96003b8997e348ed3b0870162c590fb734e5b1/app/Console/Command/AdminShell.php#L205

### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       |  RedHat
| OS version (client)       | MacOS
| PHP version               | 7.4
| MISP version / git hash   | 2.4.133
| Browser                   | If applicable

### Expected behavior

When the update object templates function discovers no out-of-date templates, no new templates, it should accurately report that the database is in sync with the file system and no updates are required.

### Actual behavior

This misleading output appears:
```
# app/Console/cake  Admin updateObjectTemplates 14

Welcome to CakePHP v2.10.22 Console
---------------------------------------------------------------
App : app
Path: /var/www/MISP/app/
---------------------------------------------------------------
Could not update object templates
```

### Steps to reproduce the behavior

Run the command shown above on a MISP system where the object templates directory has not been updated.

### Logs, screenshots, configuration dump, ...

This issue was motivated by investigating #6708

**Comment by chrisinmtown** [comment]
I propose this change:
```
diff --git a/app/Console/Command/AdminShell.php b/app/Console/Command/AdminShell.php
index e77575d56..b77f6c8fd 100644
--- a/app/Console/Command/AdminShell.php
+++ b/app/Console/Command/AdminShell.php
@@ -217,14 +217,14 @@ class AdminShell extends AppShell
                 if ($result) {
                     echo 'Object templates updated' . PHP_EOL;
                 } else {
-                    echo 'Could not update object templates' . PHP_EOL;
+                    echo 'No object templates were updated' . PHP_EOL;
                 }
             } else {
                 $result = $this->ObjectTemplate->update($user, false,false);
                 if ($result) {
-                    echo 'Object templates updated' . PHP_EOL;
+                    echo 'Object templates updated for user ' . $user . PHP_EOL;
                 } else {
-                    echo 'Could not update object templates' . PHP_EOL;
+                    echo 'No object templates were updated for user ' . $user . PHP_EOL;
                 }
             }
         }
```

**Comment by chrisinmtown** [comment]
The fix looks very good. Because it's a minor fix, and based on your new release-fixes-quickly approach published with the release of version 2.4.137, I expected this to be committed straight to branch 2.4. But instead I see it's committed to branch `develop`.  Did I misunderstand the scope of this fix and/or your fix-vs-feature branch usage?
